install_sct

```diff
@@ -77,33 +90,34 @@ function die() {
 # Run a command and display it in color. Exit if error.
 # @input: string: command to run
 function run() {
-  cmd=$1
-  print code "$cmd"
-  $cmd
-  if [[ $? != 0 ]]; then
-    die "ERROR: Command failed."
-  fi
+  ( # this subshell means the 'die' only kills this function and not the whole script;
+    # the caller can decide what to do instead (but with set -e that usually means terminating the whole script)
+    print code "$@"
+    if ! "$@" ; then
+      die "ERROR: Command failed."
+    fi
+  )
 }
 
 # Force a clean exit
 function finish() {
   # Catch the last return code
-  value=$?
+  value="$?"
   # Get back to starting point
-  cd $SCT_SOURCE
-  if [[ $value -eq 0 ]]; then
+  cd "$SCT_SOURCE"
+  if [[ "$value" -eq 0 ]]; then
     print info "Installation finished successfully!"
-  elif [[ $value -eq 99 ]]; then
+  elif [[ "$value" -eq 99 ]]; then
     # Showing usage with -h
     echo ""
   else
     print error "Installation failed!\n
-Please copy the historic of this Terminal (starting with the command install_sct) and paste it in a new created topic on SCT's forum:\n
+Please copy the output of this Terminal (starting with the command install_sct) and upload it as a .txt attachment in a new topic on SCT's forum:\n
```

Comment:
@jcohenadad please vet this wording change. I recommend it, but it doesn't have to go in.

Reply:
good idea, i was getting increasingly annoyed by the large copy/paste without respecting markdown "code" format